nimbusapp

```diff
@@ -11,7 +11,8 @@ function usage()
    echo -e "             dockerapp is added if not explicitly stated."
    echo -e "  VERSION    The version of the Docker App file you wish to run."
    echo -e "  -s, --set  Enables you to set(override) default arguments"
-   echo -e "  -v         Volume mounts .m2 and IdeaProject for access from the NimbusServer - EXPERIMENTAL NOT FOR CASUAL USERS"
```

Comment:
Nimbus app shouldn't necessarily name nimbus server specifically (nimbusapp may eventually be able to use on different machines than nimbus server).
I think the description should read "-v       Mounts volume for IdeaProject in IntelliJ container into user's home directory - EXPERIMENTAL: ONLY USE IF YOU UNDERSTAND VOLUME MOUNTS". Or something similar.
same for the -m

Reply:
Sure.  I have not problem changing your original words.  😄